### PR TITLE
Enable hyperlinks in topic text to be clicked

### LINKED ISSUE #1042: Clickable URL in room status
Hi,

Room status often shares a link (to wiki, room archives, etc.) . Dino does not make this URL clickable. Would you mind to implement such feature ?

Regards,
Étienne

### PR DESCRIPTION
**Pull request by eerielili**
should close #1042 and save some clicks

before
image

after
image

### CODE REVIEW

**Comment by licaon-kter** [comment]
But what if the url is not visible because it's further down the text?

File: main/src/ui/conversation_view_controller.vala [hunk 1]
```diff
@@ -178,10 +178,12 @@ public class ConversationViewController : Object {
     private void update_conversation_topic(string? subtitle = null) {
         if (subtitle != null) {
             conversation_topic = Util.summarize_whitespaces_to_space(subtitle);
```
Comment (fiaxh) [inline main/src/ui/conversation_view_controller.vala:180]: Please use a dedicated variable here. `conversation_topic` is synced with the actual label in the UI, and putting raw user input into a label that has `use_markup=true` can lead to issues.

File: main/src/ui/conversation_view_controller.vala [hunk 2]
```diff
@@ -178,10 +178,12 @@ public class ConversationViewController : Object {
...
+            conversation_topic = Util.parse_add_markup(conversation_topic, null, true, false);
```
Comment (fiaxh) [inline main/src/ui/conversation_view_controller.vala:181]: You could also set `parse_text_markup = true`

**Review by fiaxh — CHANGES_REQUESTED** [review]

File: main/src/ui/conversation_view_controller.vala (hunk 1 above)
Comment (eerielili) [reply main/src/ui/conversation_view_controller.vala:181]: @fiaxh done, added a `string topic_markup` at top of function

File: main/src/ui/conversation_view_controller.vala (hunk 2 above)
Comment (eerielili) [reply main/src/ui/conversation_view_controller.vala]: @fiaxh done setting `parse_text_markup = true`

**Comment by eerielili** [comment]
@licaon-kter i guess one could click the conversation details, the links are clickable there too. It's a good point, I do not know how to approach it.

**Comment by licaon-kter** [comment]
https://github.com/dino/dino/issues/962 i guess

**Review by fiaxh — APPROVED** [review]

**Pull request merged**